ph5/core/tests/test_base.py

```diff
@@ -28,9 +29,9 @@ def kef_to_ph5(ph5path, nickname, kefpath, keflist, ex=None):
 
     :para ph5path: path to ph5 file
     :type ph5path: string
-    :para kefpath: path to kef files
-    :type kefpath: string
-    :para keflist: A list of kef file names
+    :para commonpath: common part of paths to kef files
+    :type commonpath: string
+    :para keflist: A list of different parts of paths to kef files
```

Comment:
The caller can sort out the paths.

Reply:
I don't understand this comment.
In this function, if I want to add metadata from 2 kef files to master.ph5, 'ph5/test_data/metadata/array_t_9.kef' and 'ph5/test_data/rt125a/das_t_12183.kef', the commonpath param will be 'ph5/test_data' and keflist param will be ['metadata/array_t_9.kef', 'rt125a/das_t_12183.kef']. I do that so the repeat part will not be written multiple times.